components/inputs/test/input-time-range.visual-diff.js

```diff
@@ -70,7 +70,7 @@ describe('d2l-input-time-range', () => {
 		await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
 	});
 
-	describe('opened behavior', () => {
+	describe.skip('opened behavior', () => {
```

Comment:
AS discussed in standup, skipping these and opened https://github.com/BrightspaceUI/core/issues/1749